jobs/forms.py

```diff
@@ -5,10 +5,17 @@
 
 class JobForm(forms.ModelForm):
 
-    company_name = forms.CharField(max_length=150, required=True)
+    company_name = forms.CharField(label='Company',
+                                   max_length=500,
+                                   required=True)
+    website = forms.URLField(label='Company website', required=True)
 
     class Meta:
         model = Job
-        fields = ('contact_email', 'title', 'description', 'city',
-                  'country', 'expiration_date')
-        exclude = ('company',)
+        fields = ('company_name', 'website', 'contact_email', 'title',
+                  'description', 'city', 'country')
+        # custom labels
+        labels = {
+            'title': 'Job Title'
+        }
+
```

Comment:
you can specify custom labels for the model fields like this (not the exrta defined fields, though, only the ones that come directly from the model).  You can also specify custom help text and custom widgets in a similar way if you want to.  This keeps that out of the template - so you don't need to define your labels there.